src/app/content/components/CallToActionPopup/styles.ts

```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components';
+import { Times } from 'styled-icons/fa-solid/Times/Times';
+import Button from '../../../components/Button';
+import { textRegularStyle } from '../../../components/Typography';
+import { textStyle } from '../../../components/Typography/base';
+import theme from '../../../theme';
+
+// tslint:disable-next-line: variable-name
+export const CTAWrapper = styled.div`
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  z-index: 99;
+  width: 100%;
+
+  @media screen and (max-width: 1200px) {
+    display: none;
+  }
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAContent = styled.div`
+  position: absolute;
+  z-index: 3;
+  bottom: 0;
+  width: 100%;
+  display flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAText = styled.div``;
+
+// tslint:disable-next-line: variable-name
+export const CTAHeading = styled.h2`
+  ${textStyle}
+  font-size: 3.6rem;
+  line-height: 4rem;
+  color: ${theme.color.text.white};
+  margin: 0;
+  max-width: 570px;
+  overflow: hidden;
+`;
+
+// tslint:disable-next-line: variable-name
+export const CTAParagraph = styled.p`
+  ${textStyle}
+  font-size: 1.8rem;
+  line-height: 3rem;
+  margin: 1rem 0;
+  color: ${theme.color.text.white};
+  max-width: 570px;
```

Comment:
we normally use `rem`s for sizes and spacing, so that they work with different default text sizes and zooming
```suggestion
  max-width: 57rem;
```